fe/netlify.toml

```diff
@@ -3,20 +3,21 @@
 	publish = ".svelte-kit/netlify"
 
 [build.environment]
-	NODE_VERSION = "18.13.0"
 	ADAPTER = "netlify"
+	NODE_VERSION = "18.13.0"
+	VITE_BE_ENV = "dev"
 
 [functions]
 	# temporary workaround for https://github.com/sveltejs/kit/issues/6462
 	node_bundler = "esbuild"
```

Comment:
Probably not in this PR, but it has been some time since this was needed and maybe it can now be removed

Reply:
Agree! https://github.com/nestauk/asf_hp_market_tracker/issues/215